.travis.yml

```diff
@@ -6,4 +6,6 @@ python:
 install:
   - "pip install ."
   - "pip install -r requirements.txt"
-script: py.test
\ No newline at end of file
+script:
+  - py.test /sqlalchemy_jsonapi/tests/
```

Comment:
Thanks :) I made fix hopefully it will find tests. I completely forgot about ./ since in tox I don't have to worry about that.